modular-docs-manual/files/TEMPLATE_PROCEDURE_doing-one-procedure.adoc

```diff
@@ -18,6 +18,8 @@ The ID is an anchor that links to the module. Avoid changing it after the module
 = Doing procedure A
 ////
 Start the title of a procedure module with a gerund, such as Creating, Installing, or Deploying.
+
+Be sure to include a line break between the title and the module introduction.
```

Comment:
"Include" can come across as "add", which isn't needed in this case.
How about "maintain" or something similar? For example:
```suggestion
Maintain the line break between the title and module introduction.
```

Reply:
Thanks Rolfe, I used the same line that is already in the other templates for consistency. So  I guess it would be best to leave the line as it is or update the same line in the other templates too.